README.md

```diff
@@ -258,146 +273,40 @@ In general, you should never need to commit a submodule update to `terraform-web
-pushd "$GOPATH/src/github.com/hashicorp/terraform-website/content/source/layouts"
-ln -sf "../../../ext/providers/$PROVIDER_REPO/website/$PROVIDER_REPO.erb" "$PROVIDER_REPO.erb"
-popd
-
-# link the content
-pushd "$GOPATH/src/github.com/hashicorp/terraform-website/content/source/docs/providers"
-ln -sf "../../../../ext/providers/$PROVIDER_REPO/website/docs" "$PROVIDER_REPO"
-popd
-
-# start middleman
-cd "$GOPATH/src/github.com/terraform-provider-$PROVIDER_REPO"
-make website
-
-# Note: if you haven't already, copy a GNUmakefile from one of the other provider repos
-# e.g.: https://github.com/terraform-providers/terraform-provider-google/blob/master/GNUmakefile
-```
-
-- Finally, open `http://localhost:4567/docs/providers/[your-provider]` in your web browser to visualize your provider's docs.
-
 ## Finding Broken Links
 
 ↥ [back to top](#table-of-contents)
 
-Broken links are the scourge of the web, so the tooling around terraform.io includes some warning systems to help us spot and fix them. The process of working with those systems could be nicer, but here's how it works today.
-
-### Step 1: See a Failing Build
-
-There is one place that will typically warn you about broken links:
-
-- Failing CircleCI builds. CircleCI runs tests on pull requests to terraform-website, and the result is shown in the PR. Circle also deploys the website to prod, and sends success/fail messages to the #proj-terraform-docs channel in HashiCorp's Slack workspace. (This one isn't intended as a link check, but it spiders the whole site to warm up the Fastly cache, which has almost the same effect. The only real difference is that it obeys redirects, since it's hitting prod.)
+Terraform.io uses a few different link checkers, which run as CircleCI jobs. If a link checking job fails, you can go to the job in CircleCI to find out which link URL caused the problem and which page that link appeared on.
 
-In this case, the failing job usually doesn't mean the actual build or deploy failed, and instead means that the link-checking or cache-warming scripts found a broken link and exited with a non-zero status code.
+All of these jobs are configured in `./circleci/config.yml`, in this repo and in hashicorp/terraform.
 
-### Step 1a: Identify the First Bad Build
+### Global Link Check
 
-If you noticed the failing builds early enough, you might be able to see the exact point where they flipped from green to red. If so, look at the commit associated with the first bad build!
+We run a global link check for the whole site after every deploy.
 
-Knowing the commit can save you a lot of time in step 4, when you're trying to figure out which files include the broken links.
+- **Where:** The job reports its status in the `#proj-terraform-docs` channel in Hashicorp's Slack.
+- **What:** This job only checks internal links within terraform.io, not external links to the rest of the web. (We deploy a _lot,_ and we don't want to be a nuisance on someone else's servers.)
```

Comment:
Will we really deploy that much any more now that we don't deploy every time a provider release happens?

Reply:
3 deploys yesterday, 4 on wednesday, 3 on tuesday... so, several times a day. I guess it's a relative question, huh? 

```suggestion
- **What:** This job only checks internal links within terraform.io, not external links to the rest of the web. (It runs frequently, and we don't want to be a nuisance.)
```